TombEngine/Scripting/Internal/TEN/Flow/Level/FlowLevel.h

```diff
@@ -38,6 +38,7 @@ struct Level : public ScriptInterfaceLevel
 	int LevelFarView{ 0 };
 	bool UnlimitedAir{ false };
 	std::vector<InventoryItem> InventoryObjects;
+	short LevelSecrets{ 0 };
```

Comment:
I think you can make this property `int`, no need in using `short` type anywhere in new code.

Reply:
done